app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java

```diff
@@ -883,10 +876,10 @@ public void captureHighResolutionTraces(String channel, int samples, double time
 
     public int[] oscilloscopeProgress() {
         /*
-        * returns the number of samples acquired by the capture routines, and the conversion_done status
-        *
-        * return structure int[]{conversionDone, samples}
-        */
+         * returns the number of samples acquired by the capture routines, and the conversion_done status
```

Comment:
@harsh-2711 this PR looks good. But since this is mostly functional code which is really great work from you I need to go through lines clearly. So can you keep only the changes you made for now and revert these kind of reformatting and refactoring done to code parts not relevant?